policy/common/methods.go

```diff
@@ -179,3 +179,7 @@ func (m *Methods) BodyMatches(prBody string) bool {
 	}
 	return false
 }
+
+func (m *Methods) IsEmpty() bool {
+	return m == nil || (len(m.Comments) == 0 && len(m.CommentPatterns) == 0 && len(m.GithubReviewCommentPatterns) == 0 && len(m.BodyPatterns) == 0)
```

Comment:
oh i can just put it here, and maybe call this something different.

Reply:
yea i realized that, but with the new approach, i think we don't need it anymore.